apple/internal/rule_factory.bzl

```diff
@@ -920,6 +926,39 @@ ignored.
 """,
             ),
         })
+    elif rule_descriptor.product_type == apple_product_type.framework:
+        attrs.append({
+            # TODO: This attribute is not publicly documented, but it is tested in
+            # http://github.com/bazelbuild/rules_apple/test/ios_framework_test.sh?l=79. Figure out
+            # what to do with this.
+            "hdrs": attr.label_list(
+                allow_files = [".h"],
+            ),
```

Comment:
Well, since the dynamic_framework rules don't support objective c yet, I guess we don't NEED to allow headers to be passed in. I could delete this section of code along with the call to framework_headers_partial if that's the case. I originally copied it from _get_ios_attrs in rule_factory.bzl. It looks like the ios_framework rule does support it, so in a different PR we could document it and delete the TODO?